libretroshare/src/rsserver/p3peers.cc

```diff
@@ -1099,6 +1109,72 @@ bool p3Peers::GetPGPBase64StringAndCheckSum(	const RsPgpId& gpg_id,
 	return true ;
 }
 
+bool p3Peers::acceptInvite( const std::string& invite,
+                            ServicePermissionFlags flags )
+{
+	if(invite.empty()) return false;
+
+	const std::string* radixPtr(&invite);
+
+	RsUrl url(invite);
+	std::map<std::string, std::string> query(url.query());
+
+	if(query.find("radix") != query.end())
+		radixPtr = &query["radix"];
+
+	const std::string& radix(*radixPtr);
```

Comment:
Since `radixPtr` is set to be the begin of the string i assume the failure of RsUrl was taken into account. And thus using a certificate works fine.

Reply:
Yeah it was like that by design ;)